README.md

```diff
@@ -160,6 +160,11 @@ which uses musl instead.
 
 [Alpine grpc-java package]: https://pkgs.alpinelinux.org/package/edge/testing/x86_64/grpc-java
 
+If you're running on a JDK version newer than than JDK 8, you'll also need to
```

Comment:
I think you might need move this paragraph above the section **Generated Code**

Reply:
Makes sense. Modified!